runtime/src/bank.rs

```diff
@@ -71,6 +71,29 @@ use std::{
     sync::{Arc, RwLock, RwLockReadGuard},
 };
 
+// Partial SPL Token v2.0.x declarations inlined to avoid an external dependency on the spl-token crate
+pub mod inline_spl_token_v2_0 {
+    solana_sdk::declare_id!("TokenkegQfeZyiNwAJbNbGKPFXCWuBvf9Ss623VQ5DA");
+    pub mod native_mint {
+        solana_sdk::declare_id!("So11111111111111111111111111111111111111112");
+
+        /*
+            Mint {
+                mint_authority: COption::None,
+                supply: 0,
+                decimals: 9,
+                is_initialized: true,
+                freeze_authority: COption::None,
+            }
+        */
+        pub const ACCOUNT_DATA: [u8; 82] = [
+            0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
I assume this is correct. Or should I check this too? :)

Reply:
Thanks, I'm quite sure it's correct.   "Trust me" ;)